.github/workflows/docker-image-push.yml

```diff
@@ -0,0 +1,58 @@
+name: Build (and push) multi-arch Docker image
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    # We want a newer version of qemu: https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1815100
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        dockerfile: [Dockerfile, alpine/Dockerfile]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Docker login
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
```

Comment:
Shouldn't this `if` be moved to `jobs.build`, like line 9?

Reply:
We want to build for all commits but only push for tags.